meshio/helpers.py

```diff
@@ -76,7 +76,24 @@
     '.vtk': 'vtk-binary',
     '.xdmf': 'xdmf',
     '.xmf': 'xdmf',
-    }
+}
+
+
+def check_filename(filename, file_format=None):
+    # Checks the input and output file name and format.
+    # https://stackoverflow.com/q/4843173/353337
+    assert isinstance(filename, str), 'No such file or directory: {}'.format(filename)
```

Comment:
The error message is misleading, it should be "Is no string". Not sure if this is needed at all.

Reply:
Sure. I don't know anymore why this assertion was inserted at all though. Anyhow, let's keep it. Perhaps the input can also be shown in the error message (à la `Invalid input '123'.`)